_automation/main.go

```diff
@@ -95,6 +103,8 @@ func root(args []string) error {
 		flagsParse(fs, args[1:])
 
 		s.UpdateAll(ctx)
+	case "run-tests":
```

Comment:
I currently just use `go test -v ./...` to test all the grammars after update. What is the advantage of running them from this script instead?

Reply:
That's fine, this can be dropped. I'll update the readme to simply instruct user to run `go test -v ./...`.

Reply:
That's fine, this can be dropped. I'll update the readme to simply instruct user to run `go test -v ./...`.